core/test-extension/deployment/src/test/java/io/quarkus/extest/ConfiguredBean.java

```diff
@@ -1,20 +1,19 @@
 package io.quarkus.extest;
 
-import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.event.Observes;
-import javax.inject.Inject;
+import javax.enterprise.inject.spi.CDI;
```

Comment:
I don't see this import used (just checking on GH so I might be missing something)